gen/testdata/thrift/constants.thrift

```diff
@@ -0,0 +1,86 @@
+include "./other_constants.thrift"
+include "./containers.thrift"
+include "./enums.thrift"
+include "./exceptions.thrift"
+include "./structs.thrift"
+include "./unions.thrift"
+
+const containers.PrimitiveContainers primitiveContainers = {
+    "listOfInts": other_constants.listOfInts, // imported constant
```

Comment:
github seems to be unhappy with `// ` comments in this file. should we be using something else?